utils/scapp.h

```diff
@@ -389,15 +493,21 @@ class scApp : public ktxApp {
           "               deterministic).\n\n"
           "  --no_sse     Forbid use of the SSE instruction set. Ignored if CPU does not\n"
           "               support SSE. Only the Basis Universal compressor uses SSE.\n"
-          "  --verbose    Print encoder/compressor activity status to stdout. Currently\n"
-          "               only the Basis Universal compressor emits status.\n"
           "  --zcmp [<compressionLevel>]\n"
           "               Supercompress the data with Zstandard. Implies --t2. Can be used\n"
-          "               with data in any format except ETC1S / BasisLZ (--bcmp). Most\n"
+          "               with data in any format except ETC1S / BasisLZ (bcmp). Most\n"
           "               effective with RDO-conditioned UASTC or uncompressed formats. The\n"
           "               optional compressionLevel range is 1 - 22 and the default is 3.\n"
           "               Lower values=faster but give less compression. Values above 20\n"
-          "               should be used with caution as they require more memory.\n";
+          "               should be used with caution as they require more memory.\n"
+          "  --threads <count>\n"
+          "               Explicitly set the number of threads to use during compression.\n"
+          "               By default, ETC1S / BasisLZ and ASTC compression will use the number of\n"
+          "               threads reported by thread::hardware_concurrency or 1 if value\n"
+          "               returned is 0.\n"
+          "  --verbose    Print encoder/compressor activity status to stdout. Currently\n"
+          "               only both astcencoder and Basis Universal compressor emits status.\n"
```

Comment:
s/only both astcencoder and Basis Universal compressor emits/only the astc, etc1s and uastc encoders emit/

Let's settle on encoder for everything except zstd for which we'll use compressor.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software/pull/433/commits/c01d05d4cfa0353f848c18051d90ba94862115b5